### PR TITLE
Sync metrics and minor tweaks

### PR DESCRIPTION
**Pull request by marlycormar**
- Set `riskmetric` version to **release-v0.1.1**.
- Add metrics **r_cmd_check** and **remote_checks** to match `riskmetric`. These metrics have been added with weight 0.

Related to #142.

### CODE REVIEW

File: global.R [hunk 1]
```diff
@@ -42,11 +42,9 @@ package.check <- lapply(
   }
 )
 
-# Install riskmetric from github and load it
-if(!require(riskmetric)){
-  devtools::install_github("pharmaR/riskmetric")
-  library(riskmetric)
-}
+# Install a specific version of riskmetric from GitHub and load it.
+devtools::install_github("pharmaR/riskmetric", ref = "release-v0.1.1")
```
Comment (AARON-CLARK) [inline global.R:46]: So, this line of code will re-install the `0.1.1` version every time you run the app. Is that the intended behavior? How about we use an if-then statement using packageVersion("riskmetric") to see if the user has the correct version. We can also add this version to the `DESCRIPTION` file.

image

**Review by AARON-CLARK — CHANGES_REQUESTED** [review]
Hi @marlycormar, comments below! Plus I'll push a commit to import this version of `riskmetric`. Stay tuned.

File: global.R (hunk 1 above)
Comment (marlycormar) [reply global.R:47]: Change this line by `library(riskmetric)`. The package version of the app will take care of package versioning.

Comment (marlycormar) [reply global.R:47]: Hi @AARON-CLARK, I am having trouble pushing the updated `global.R file`. My local git doesn't know how to reconcile my changes with the branches that are now part of this PR. 

Could you please undo the changes I made? Basically remove lines 46-47 and add back 45-49?

Comment (AARON-CLARK) [reply global.R]: done!

**Review by AARON-CLARK — APPROVED** [review]
LGTM!

**Comment by AARON-CLARK** [comment]
Hey @marlycormar, this branch looking good. Did you have any other changes planned or can I merge? I'd like to start "packagizing" the app after these changes are pushed through!

**Comment by marlycormar** [comment]
Yes, please.

**Pull request merged**